doc/programming/samples.rst

```diff
@@ -150,21 +150,53 @@ This code sample simulates the passive dynamics of a given model, renders it off
 values, and saves them into a raw data file that can then be converted into a movie file with tools such as ffmpeg. The
 rendering is simplified compared to :ref:`simulate.cc <saSimulate>` because there is no user interaction, visualization
 options or timing; instead we simply render with the default settings as fast as possible. The dimensions and number of
-multi-samples for the offscreen buffer are specified in the MuJoCo model, while the simulation duration, frames-per-
-second to be rendered (usually much less than the physics simulation rate), and output file name are specified as
-command-line arguments. For example, a 5 second animation at 60 frames per second is created with:
+multi-samples for the offscreen buffer are specified in the MuJoCo model with the visual/global/{offwidth,offheight}
```

Comment:
No. 

multisamples are specified with 
https://mujoco.readthedocs.io/en/stable/XMLreference.html#visual-quality-offsamples
height and width are specified with
https://mujoco.readthedocs.io/en/stable/XMLreference.html#visual-global-offheight
https://mujoco.readthedocs.io/en/stable/XMLreference.html#visual-global-offwidth

Reply:
(Of course use proper RST links, not raw html)

Reply:
Great! Is this what you had in mind? :)
https://github.com/google-deepmind/mujoco/blob/f2873724672e660e6b095384ba54e3a499cd3447/doc/programming/samples.rst#record

Reply:
Perfect :)